### PR TITLE
ANY23-450 Update Maven deps and plugin versions

### PR DESCRIPTION
**Pull request by lewismc**
This is a simple batch dep and plugin update... verified that it does not break any unit tests with mvn clean install

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -381,176 +381,176 @@
       <dependency>
         <groupId>org.apache.tika</groupId>
         <artifactId>tika-core</artifactId>
-        <version>${tika.version}</version>
+        <version>1.24</version>
```
Comment (HansBrende) [inline pom.xml]: Can we update the version declared in properties instead?

File: pom.xml [hunk 2]
```diff
@@ -381,176 +381,176 @@
...
         <groupId>org.apache.poi</groupId>
         <artifactId>poi-scratchpad</artifactId>
-        <version>${poi.version}</version>
+        <version>4.1.2</version>
       </dependency>
       <dependency>
         <groupId>com.fasterxml.jackson.core</groupId>
         <artifactId>jackson-core</artifactId>
-        <version>${jackson.version}</version>
+        <version>2.11.0.rc1</version>
```
Comment (HansBrende) [inline pom.xml]: Let's not use release candidates... I prefer to stay on stable versions only.

File: pom.xml [hunk 3]
```diff
@@ -381,176 +381,176 @@
...
         <groupId>com.github.jsonld-java</groupId>
         <artifactId>jsonld-java</artifactId>
-        <version>0.12.5</version>
+        <version>0.13.0</version>
       </dependency>
       <dependency>
         <groupId>org.semarglproject</groupId>
         <artifactId>semargl-rdf4j</artifactId>
-        <version>${semargl.version}</version>
+        <version>0.7</version>
```
Comment (HansBrende) [inline pom.xml]: Don't know why we replaced the version declared in the properties section.

File: pom.xml [hunk 4]
```diff
@@ -565,49 +565,49 @@
...
         <groupId>org.slf4j</groupId>
         <artifactId>jul-to-slf4j</artifactId>
-        <version>${slf4j.logger.version}</version>
+        <version>2.0.0-alpha1</version>
       </dependency>
       <dependency>
         <groupId>org.slf4j</groupId>
         <artifactId>slf4j-log4j12</artifactId>
-        <version>${slf4j.logger.version}</version>
+        <version>2.0.0-alpha1</version>
```
Comment (HansBrende) [inline pom.xml]: Prefer to stay away from alpha versions...

**Comment by lewismc** [comment]
Thanks for the review @HansBrende 
I've addressed your concerns.

**Comment by HansBrende** [comment]
@lewismc great! Is there another commit? I see only the original commit.

**Comment by lewismc** [comment]
Sorry, please check now @HansBrende

File: pom.xml [hunk 5]
```diff
@@ -329,37 +329,37 @@
...
         <groupId>org.apache.commons</groupId>
         <artifactId>commons-lang3</artifactId>
-        <version>3.9</version>
+        <version>3.10</version>
       </dependency>
       <dependency>
         <groupId>org.apache.httpcomponents</groupId>
         <artifactId>httpclient</artifactId>
-        <version>${httpclient.version}</version>
+        <version>4.5.12</version>
```
Comment (HansBrende) [inline pom.xml]: This one still needs to be updated in properties and referenced by `${httpclient.version}`

File: pom.xml [hunk 6]
```diff
@@ -329,37 +329,37 @@
...
       </dependency>
       <dependency>
         <groupId>org.apache.httpcomponents</groupId>
         <artifactId>httpclient-cache</artifactId>
-        <version>${httpclient.version}</version>
+        <version>4.5.12</version>
```
Comment (HansBrende) [inline pom.xml]: Ditto

File: pom.xml [hunk 7]
```diff
@@ -329,37 +329,37 @@
...
       </dependency>
       <dependency>
         <groupId>org.apache.httpcomponents</groupId>
         <artifactId>httpmime</artifactId>
-        <version>${httpclient.version}</version>
+        <version>4.5.12</version>
```
Comment (HansBrende) [inline pom.xml]: Ditto

File: pom.xml [hunk 8]
```diff
@@ -329,37 +329,37 @@
...
       </dependency>
       <dependency>
         <groupId>org.apache.httpcomponents</groupId>
         <artifactId>httpcore</artifactId>
-        <version>${httpcore.version}</version>
+        <version>4.4.13</version>
```
Comment (HansBrende) [inline pom.xml]: Ditto

File: pom.xml [hunk 9]
```diff
@@ -391,17 +391,17 @@
       <dependency>
         <groupId>org.apache.poi</groupId>
         <artifactId>poi</artifactId>
-        <version>${poi.version}</version>
+        <version>4.1.2</version>
```
Comment (HansBrende) [inline pom.xml]: Ditto

File: pom.xml [hunk 10]
```diff
@@ -391,17 +391,17 @@
...
       </dependency>
       <dependency>
         <groupId>org.apache.poi</groupId>
         <artifactId>poi-ooxml</artifactId>
-        <version>${poi.version}</version>
+        <version>4.1.2</version>
```
Comment (HansBrende) [inline pom.xml]: Ditto

File: pom.xml [hunk 11]
```diff
@@ -391,17 +391,17 @@
...
       </dependency>
       <dependency>
         <groupId>org.apache.poi</groupId>
         <artifactId>poi-scratchpad</artifactId>
-        <version>${poi.version}</version>
+        <version>4.1.2</version>
```
Comment (HansBrende) [inline pom.xml]: Ditto

File: pom.xml [hunk 12]
```diff
@@ -424,92 +424,92 @@
       <dependency>
         <groupId>org.eclipse.rdf4j</groupId>
         <artifactId>rdf4j-model</artifactId>
-        <version>${rdf4j.version}</version>
+        <version>3.1.2</version>
```
Comment (HansBrende) [inline pom.xml]: Ditto for all the rdf4j dependencies below

File: pom.xml [hunk 13]
```diff
@@ -522,35 +522,35 @@
       <dependency>
         <groupId>net.sourceforge.owlapi</groupId>
         <artifactId>owlapi-api</artifactId>
-        <version>${owlapi.version}</version>
+        <version>5.1.13</version>
```
Comment (HansBrende) [inline pom.xml]: Ditto for all the owlapi deps

File: pom.xml [hunk 14]
```diff
@@ -522,35 +522,35 @@
...
         <groupId>org.apache.commons</groupId>
         <artifactId>commons-csv</artifactId>
-        <version>1.7</version>
+        <version>1.8</version>
       </dependency>
       <dependency>
         <groupId>commons-io</groupId>
         <artifactId>commons-io</artifactId>
-        <version>${commons-io.version}</version>
+        <version>2.6</version>
```
Comment (HansBrende) [inline pom.xml]: Ditto

File: pom.xml [hunk 15]
```diff
@@ -349,17 +349,17 @@
       <dependency>
         <groupId>org.apache.httpcomponents</groupId>
         <artifactId>httpcore</artifactId>
-        <version>${httpcore.version}</version>
+        <version>4.4.13</version>
```
Comment (HansBrende) [inline pom.xml:352]: Last one: need to update httpcore.version to 4.4.13 and change this to ${httpcore.version}

**Comment by lewismc** [comment]
Boom. Thanks v much @HansBrende for review.

**Pull request merged**